src/components/navigation/Tab.tsx

```diff
@@ -19,7 +19,14 @@ type ComponentProps = {
 
 type HTMLAttributes = Omit<
   JSX.HTMLAttributes<HTMLButtonElement>,
-  'size' | 'icon' | 'title'
+  | 'size'
+  | 'icon'
+  | 'title'
+  | 'selected'
+  // Omitting these aria attributes, as they are set as `never` in ButtonBase
+  | 'aria-expanded'
+  | 'aria-pressed'
+  | 'aria-label'
```

Comment:
Preact 10.13 introduces these aria attributes in the type, making it incompatible with `ButtonBase` props, where they are all explicitly set as `never` to indicate there are alternative props to be used.

I'm wondering if we should omit them there instead. It should have the same effect as `never`. but I'm not sure if we could introduce some other side effect.